pwnlib/tubes/process.py

```diff
@@ -345,6 +357,9 @@ def __init__(self, argv = None,
 
             p.success('pid %i' % self.pid)
 
+        if context.os == "windows":
+            return None
+
```

Comment:
explicit `None` should be avoided in methods not returning anything; what breaks in the further code path?